### PR TITLE
ECR Mirror

### PR DESCRIPTION
**Pull request by apahim**
Signed-off-by: Amador Pahim <apahim@redhat.com>

### CODE REVIEW

File: reconcile/ecr_mirror.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+            account=account,
+            region=region,
+        )
+        self.ecr_auth = f'{self.ecr_username}:{self.ecr_password}'
+
+        self.image_username = None
+        self.image_password = None
+        self.image_auth = None
+        push_secret = self.instance['mirror']['pullCredentials']
+        if push_secret is not None:
```
Comment (jmelis) [inline reconcile/ecr_mirror.py:59]: why this if?

Comment (jmelis) [reply reconcile/ecr_mirror.py:59]: if it so happens that push_secret is None, shouldn't we raise an exception?

Comment (apahim) [reply reconcile/ecr_mirror.py:59]: it's pull secret... typo. Repo to pull from might be public.

Comment (jmelis) [reply reconcile/ecr_mirror.py:59]: nevermind, I understand it now

**Review by jmelis — APPROVED** [review]
LGTM

**Pull request merged**